packages/design-system/src/components/analytics/Analytics.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Functions for sending events to Tealium/Google Analytics
+ * Based on HRA Tool & SEP screener & Coverage Tools analytics service:
+ * - hra-tool/src/services/analytics.js
+ * - screener/src/assets/js/utils/analytics-util.js
+ * - coverage-tools-frontend/src/helpers/objectUtilities.ts
+ */
+
+/* Extend the global Window interface to fix ts error TS2339 */
+declare global {
+  interface Window {
+    utag?: {
+      link: (params: AnalyticsPayload) => void;
+    };
+  }
+}
+
+type EventType = 'link';
+const MAX_RETRIES = 3;
+const TIMEOUT = 300;
+
+/* eslint-disable camelcase */
+export interface AnalyticsPayload {
+  ga_eventType: string;
+  ga_eventCategory: string;
+  ga_eventAction: string;
+  ga_eventLabel: string;
+  ga_eventValue: string;
+  additional_props?: Record<string, unknown>;
+}
+
+export const EVENT_CATEGORY = {
+  contentTools: 'content tools',
+};
+
+export const EVENT_ACTION = {
+  helpDrawerOpen: 'opened help drawer',
+  helpDrawerClose: 'closed help drawer',
+};
+
+interface AnalyticsEventProps {
+  ga_eventType: string;
+  ga_eventCategory: string;
+  ga_eventAction: string;
+  ga_eventLabel: string;
+  ga_eventValue: string;
+  additional_props?: Record<string, unknown>;
+}
+
+function sendEvent(event: EventType, props: AnalyticsPayload, retry = 0): void {
+  if (window.utag && window.utag[event]) {
+    try {
+      window.utag[event](props);
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.log('Error sending event to Tealium', e);
```

Comment:
We want to make sure this doesn't get logged out in production.

Reply:
Updated to return string instead of console.log